paddlenlp/transformers/model_utils.py

```diff
@@ -44,6 +46,38 @@
 ]
 
 
+def get_path_from_url(url: str,
+                      root_dir: str,
+                      md5sum: Optional[str] = None,
+                      check_exist: bool = True) -> str:
+    """construct `get_path_from_url` for `model_utils` to avoid multi-process downloading
```

Comment:
i am confused. This says `avoid multi-process downloading` but the title of the PR says `enable multi-process downloading`